app/views/courses/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<p id="notice"><%= notice %></p>
```

Comment:
Don't delete this file but just remove the content for now. I don't want to keep code that we're definitely not going to use.